janusgraph-test/src/main/java/org/janusgraph/diskstorage/IDAuthorityTest.java

```diff
@@ -52,43 +51,42 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
+import java.util.stream.Stream;
 
 /**
  * @author Matthias Broecheler (me@matthiasb.com)
  */
-@RunWith(Parameterized.class)
 public abstract class IDAuthorityTest {
 
     private static final Logger log =
-            LoggerFactory.getLogger(IDAuthorityTest.class);
+        LoggerFactory.getLogger(IDAuthorityTest.class);
 
     public static final int CONCURRENCY = 8;
     public static final int MAX_NUM_PARTITIONS = 4;
     public static final String DB_NAME = "test";
 
     public static final Duration GET_ID_BLOCK_TIMEOUT = Duration.ofMillis(300000L);
 
-    @Parameterized.Parameters
-    public static Collection<Object[]> configs() {
-        final List<Object[]> configurations = new ArrayList<>();
+    public static Stream<Arguments> configs() {
```

Comment:
Move first two lines from `setUp` method to a separate method, so that it would look something like:
```
public static Stream<Arguments> configs() {
    final List<Arguments> configurations = new ArrayList<>();

    ModifiableConfiguration c = getBasicConfig();
    configurations.add(arguments(checkAndReturnWriteConfiguration(c)));

    c = getBasicConfig();
    c.set(IDAUTHORITY_CAV_BITS,9);
    c.set(IDAUTHORITY_CAV_TAG,511);
    configurations.add(arguments(checkAndReturnWriteConfiguration(c)));

    c = getBasicConfig();
    c.set(IDAUTHORITY_CAV_RETRIES,10);
    c.set(IDAUTHORITY_WAIT, Duration.ofMillis(10L));
    c.set(IDAUTHORITY_CAV_BITS,7);
    //c.set(IDAUTHORITY_RANDOMIZE_UNIQUEID,true);
    c.set(IDAUTHORITY_CONFLICT_AVOIDANCE, ConflictAvoidanceMode.GLOBAL_AUTO);
    configurations.add(arguments(checkAndReturnWriteConfiguration(c)));

    return configurations.stream();
}

private WriteConfiguration checkAndReturnWriteConfiguration(ModifiableConfiguration baseConfig){
    WriteConfiguration writeConfiguration = baseConfig.getConfiguration();
    Preconditions.checkNotNull(writeConfiguration);
    TestGraphConfigs.applyOverrides(writeConfiguration);
    return writeConfiguration;
}

private void setUp(WriteConfiguration baseConfig) throws Exception {
    this.baseStoreConfiguration = baseConfig;
    Configuration config = StorageSetup.getConfig(baseConfig);
    uidBitWidth = config.get(IDAUTHORITY_CAV_BITS);
    //hasFixedUid = !config.get(IDAUTHORITY_RANDOMIZE_UNIQUEID);
    hasFixedUid = !ConflictAvoidanceMode.GLOBAL_AUTO.equals(config.get(IDAUTHORITY_CONFLICT_AVOIDANCE));
    hasEmptyUid = uidBitWidth==0;
    blockSize = config.get(IDS_BLOCK_SIZE);
    StoreManager m = openStorageManager();
    m.clearStorage();
    m.close();
    open();
}
```